app/SmartFlowUI/frontend/Pages/Admin.razor.cs

```diff
@@ -0,0 +1,97 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+namespace ClientApp.Pages;
+
+public sealed partial class Admin : IDisposable
+{
+    private MudForm _form = null!;
+
+    private bool _isLoadingProfiles = false;
+    private ProfileInfo _profileInfo = new ProfileInfo();
+    private string _profileRawData = string.Empty;
+
+    // Store a cancelation token that will be used to cancel if the user disposes of this component.
+    private readonly CancellationTokenSource _cancellationTokenSource = new();
+
+    [Inject] public required ApiClient Client { get; set; }
+    [Inject] public required ISnackbar Snackbar { get; set; }
+    [Inject] public required ILogger<Docs> Logger { get; set; }
+    [Inject] public required IJSRuntime JSRuntime { get; set; }
+    [Inject] public required HttpClient httpClient { get; set; }
+
+    protected override void OnInitialized()
+    {
+    }
+    protected override async Task OnAfterRenderAsync(bool firstRender)
+    {
+        await base.OnAfterRenderAsync(firstRender);
+        if (firstRender)
+        {
+            //showInfo("First render...  fetching profiles...");
+            await GetProfileInfoAsync();
+            StateHasChanged();
+        }
+    }
+
+    private async Task GetProfileInfoAsync()
+    {
+        _ = await Task.FromResult(true);
```

Comment:
what's this one for?

Reply:
Good catch - I had that in there was I was calling a non-async method so that the compiler didn't throw a "no await" found warning.  Now that I have another await I'll remove it.